sceptre/hooks/asg_scaling_processes.py

```diff
@@ -0,0 +1,94 @@
+# -*- coding: utf-8 -*-
+
+from sceptre.hooks import Hook
+from sceptre.exceptions import InvalidHookArgumentTypeError
+from sceptre.exceptions import InvalidHookArgumentSyntaxError
+from sceptre.exceptions import InvalidHookArgumentValueError
+
+
+class ASGScalingProcesses(Hook):
+    """
+    A command to resume or suspend autoscaling group scaling processes. This is
+    useful as scheduled actions must be suspended when updating stacks with
+    autoscaling groups.
+    This command supports all autoscaling scaling processes.
+    Example configuration:
```

Comment:
Maybe move the example to the sceptre docs  - but include note about argument syntax `!asg_scaling_processes <suspend|resume>::<process-name>`?